### DISCUSSION TITLE
Different loop templates on the same grid

### DISCUSSION THREAD

**Discussion by BarBoost** [elementor/elementor#21514] [Feature Request]
### Prerequisites

- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.

### What problem is your feature request going to solve? Please describe.

When using different Post Formats on the same grid, they all have to display with the same loop. For example, a loop that contains text and a loop that contains video must use the same template.

### Describe the solution you'd like

Allowing to assign different loops to [WordPress Post Format](https://wordpress.com/support/posts/post-formats/) and display them on the same grid.

### Describe alternatives you've considered

With WP Grid Builder, you can do exactly that (screenshot attached).

### Here's how it works in WP Grid Builder
<br><br>
### Select Post Format on the WordPress editor:
![wordpress-post-format-options](https://user-images.githubusercontent.com/113690127/222423295-2c428200-73ae-4642-a49f-2db2af6f2c89.jpg)
<br><br>
### Set different loop templates (here are called cards) to different Post Formats. From WP Grid Builder:
![Gridbuilder](https://user-images.githubusercontent.com/113690127/222424530-4727d824-dbb2-41d4-b103-cc9ef3795612.PNG)
<br><br>
### Result, Different loops in the same grid:
![grid](https://user-images.githubusercontent.com/113690127/222426724-891a3dda-da24-4c4a-8809-f27ca2950c47.jpg)
**_*Here, the loops have a similar design, but the point is that you can make a completely different design for each loop._**

**Comment by fabigrafo** [comment]
This comes with loop „alternate templates“.

**Reply by BarBoost** [reply]
What do you mean? In the Loop Grid widget, I can choose only one loop template. Am I missing something?

**Reply by fabigrafo** [reply]
In the next versions it comes. :)

**Reply by Michaelgimii** [reply]
I hope **alternate templates** will be added to loop carousel in addition to loop grid.

**Reply by fabigrafo** [reply]
> I hope **alternate templates** will be added to loop carousel in addition to loop grid.

Yes ist will 🤗
